### PR TITLE
Add More Information, FAQ, cleanup proxy config

### PR DESCRIPTION
**Pull request by user411**
Centralize initialization of common-ts / wallet-ts. Add FAQ and More Info text and links. Show About icon in header at lowest resolution.

### CODE REVIEW

**Comment by Christewart** [comment]
Here is what the rendered html looks like

Screenshot from 2022-01-27 14-48-37
Screenshot from 2022-01-27 14-49-36

File: wallet-server-ui/src/app/component/about/about.component.html [hunk 1]
```diff
@@ -36,6 +36,81 @@ <h2 translate>about.heading</h2>
...
+        A DLC Contract is an agreement between counterparties on what will happen with pooled funds based on the outcome of an event, as attested to by an oracle.
+        DLC Contracts come in two types, enumerated (enum) and numeric.
+      </p>
+      <p class="question">What is an oracle?</p>
+      <p>
+        An oracle is a trusted third party that will announce and attest to the outcome of events.
+        At the maturation time of an event, the oracle is expected to publish the outcome to the event so DLCs based on the event can resolve.
+        An example of an oracle is a cryptocurrency exchange that publishes announcements and attestations to the end-of-day price on bitcoin.
+      </p>
+      <p class="question">What is an oracle announcement?</p>
```
Comment (nkohen) [inline wallet-server-ui/src/app/component/about/about.component.html:58]: probably also what "What is an oracle attestation"

File: wallet-server-ui/src/app/component/about/about.component.html [hunk 2]
```diff
@@ -36,6 +36,81 @@ <h2 translate>about.heading</h2>
...
+      <p>
+        An announcement is a formal declaration of an event that will take place, to be attested to by a given oracle.
+        An announcement has an eventId to specify what event it is attesting to, a maturation time, and validation information for the oracle so the outcome can only be confirmed by that oracle.
+        An announcement includes the potential outcomes or range of potential outcomes depending on whether it is for an enum or numeric contract.
+      </p>
+      <p class="question">What is a contract info?</p>
+      <p>
+        A contract info is an Announcement with additional metadata including collateral and payout amounts.
+        A contract info is essentially a prototype Offer that you can review and change to make your own Offers.
```
Comment (nkohen) [inline wallet-server-ui/src/app/component/about/about.component.html:67]: I don't know that a noob would be able to parse this. Maybe something more like a contract info contains all of the information needed to construct a contract between two parties that can be turned into a DLC

**Comment by Christewart** [comment]
I didn't get any meaningful feedback on twitter besides this from ben https://twitter.com/benthecarman/status/1486806511523405826

**Comment by user411** [comment]
Yeah, I figured out the "contract contract" part. It won't be in the next revision.

**Comment by user411** [comment]
Current state
Screen Shot 2022-01-28 at 8 51 05 AM

**Comment by Christewart** [comment]
> Current state Screen Shot 2022-01-28 at 8 51 05 AM

A couple things

1. The oracle attesation section looks like it got truncated
2. For announcements, attesations, and contract infos I would add a final sentence that says "Here is an example" and link out the a good example on the OE

**Comment by user411** [comment]
Screen Shot 2022-01-28 at 9 33 16 AM

**Review by Christewart — APPROVED** [review]

**Pull request merged**